From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: c4pp4
Date: Sat, 17 Oct 2026 12:00:00 +0200
Subject: [PATCH 1/1] Remove unused libnotify

Notifications are never sent, so don't require and initialize libnotify.

Signed-off-by: c4pp4
---
 unity-indicator-appearance | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/unity-indicator-appearance b/unity-indicator-appearance
--- a/unity-indicator-appearance
+++ b/unity-indicator-appearance
@@ -4,7 +4,6 @@
 
 gi.require_version("Gtk", "3.0")
 gi.require_version('AppIndicator3', '0.1')
-gi.require_version('Notify', '0.7')
 
 import os
 import signal
@@ -18,7 +17,6 @@
 from gi.repository import Gtk as gtk
 from gi.repository import Gio as gio
 from gi.repository import AppIndicator3 as appindicator
-from gi.repository import Notify as notify
 
 APPINDICATOR_ID = 'unity-indicator-appearance'
 
@@ -31,7 +29,6 @@
     update_menu_status(appearance_settings, None)
     appearance_settings.connect('changed::gtk-theme', update_menu_status)
     appearance_settings.connect('changed::icon-theme', update_menu_status)
-    notify.init(APPINDICATOR_ID)
     gtk.main()
 
 def update_menu_status(_1, _2):
-- 
2.52.0

